lib/at_host/at_host.c

```diff
@@ -233,10 +233,12 @@ static int at_uart_init(char *uart_dev_name)
 		if (err) {
 			if (k_uptime_get_32() - start_time >
 			    CONFIG_AT_HOST_UART_INIT_TIMEOUT) {
+				LOG_ERR("UART check failed: %d. "
+				"AT host UART initialization timed out.", err);
```

Comment:
Fix indentation, if the line is too long you could omit: `AT host` since the log output is prefixed with the module name.

Reply:
Done